src/webots/nodes/utils/WbUrl.cpp

```diff
@@ -286,6 +286,8 @@ QString WbUrl::combinePaths(const QString &rawUrl, const QString &rawParentUrl)
   }
 
   if (QDir::isRelativePath(url)) {
+    if (parentUrl.startsWith(WbStandardPaths::cachedAssetsPath()))
```

Comment:
It would be more consistent to use
```suggestion
    if (WbFileUtil::isLocatedInDirectory(parentUrl, WbStandardPaths::cachedAssetsPath()))
```